typings/raf-throttle.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for raf-throttle 1.0
```

Comment:
I think I removed this dependency, so should probably remove this typings file, too